src/main/java/com/desiato/puresynth/services/AuthenticationService.java

```diff
@@ -0,0 +1,91 @@
+package com.desiato.puresynth.services;
+
+import com.desiato.puresynth.dtos.AuthenticationRequestDTO;
+import com.desiato.puresynth.dtos.PureSynthToken;
+import com.desiato.puresynth.models.CustomUserDetails;
+import com.desiato.puresynth.models.Session;
+import com.desiato.puresynth.models.User;
+import com.desiato.puresynth.repositories.SessionRepository;
+import com.desiato.puresynth.repositories.UserRepository;
+import jakarta.transaction.Transactional;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+import java.util.UUID;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Service
+public class AuthenticationService {
```

Comment:
Fields should be `final` and you should use constructor here.

Reply:
done